src/Model/BaseBlock.php

```diff
@@ -170,14 +174,32 @@ public function getUpdatedAt(): ?\DateTime
         return $this->updatedAt;
     }
 
+    public function addChild(BlockInterface $child): void
+    {
+        $this->children[] = $child;
+
+        $child->setParent($this);
+    }
+
+    /**
+     * NEXT_MAJOR: Remove this method.
```

Comment:
do we use addChild on page bundle?

Reply:
No.

In the same way we have `addChildren` method in the PageInterface of PageBundle https://github.com/sonata-project/SonataPageBundle/blob/fed26c06856de8d6b1161d0f23001db7b8377e2b/src/Model/PageInterface.php#L189 which is not used. We could rename this to `addChild`.

If you had in mind to remove the addChild method. I'm not chocked having some add/set method in the interface/class which will only be used by the user.